tavern/tomes/file_tree/metadata.yml

```diff
@@ -8,3 +8,7 @@ paramdefs:
   type: string
   label: File path
   placeholder: "/etc/"
+- name: depth
+  type: int
```

Comment:
I believe we only support type string on the backend rn :sob:

Reply:
https://github.com/spellshift/realm/blob/88fcaa4d9f5e05a9e41e17c883ae54820b628bf1/tavern/tomes/port_scan/metadata.yml#L19

Stole this from your code....